text/0000-selecting-default-java-version.md

```diff
@@ -0,0 +1,34 @@
+# Selecting the next default Java Version for Paketo Buildpacks
+
+## Summary
+
+This RFC introduces a process to select the next default version of the Paketo Java Buildpack.
+
+## Motivation
+
+Currently, there is no process how the default version of Java is picked in the Paketo world. As of today (January 2023), the default still is Java 11 (released September 2018) although Java 17 (released September 2021) is already available.
+
+## Detailed Explanation
+
+The default should be changed according to a defined process rather than at some undefined point in time to some undefined version. To do so, the default should be changed once the latest releases LTS version of Java is at least one year old. 
+
+## Rationale and Alternatives
+
+- Do nothing, update default manually from time to time
+- Always use the latest LTS immediately
+  - Could break people immediately
+- This RFC
+  - Gives room to adapt to breaking changes
+
+## Implementation
+
+To fulfill this RFC, the default version should be changed to Java 17. All future versions should be bumped according to this RFC.
+
+## Prior Art
+
+n/a
+
+## Unresolved Questions and Bikeshedding
```

Comment:
Two more questions:

- Do we want to allow for exceptions to the "at least one year" rule?

I'm not sure what/if that would be necessary, but right now there is no provision to bump the default early, if something exceptional happened.

We could add something like, "If there is a critical reason and a majority vote of all Java contributors and maintainers passes then the LTS version may be switched after 6 months time, but no earlier."

- Do we want to require an RFC to make the switch?

I think this would be more of a formality & to help with alerting folks of the change. What do folks think about this? Is it overkill? Or helpful?

Reply:
I think there should be a provision to override the one year rule and a majority vote seems reasonable. 

I don't feel an RFC is necessary.  Java contributors can be notified via slack or blogs of the upcoming change.

Reply:
> Is it overkill?

I think so. If a majority vote of all Java maintainers is in favour because of some good reason, why not be pragmatic and change it? The process should more describe the "normal" process, imho. There ca always be exceptions.

Reply:
I agree with @kevin-ortega and and @c0d1ngm0nk3y. Added a sentence to the RFC. Would that be sufficient?